### ISSUE TITLE
QUEUE TIME: counts shuffled or playlist queue

### ISSUE THREAD

**Issue by Nightmare987** [Theblockbuster1/spicetify-extensions#45]
the counter uncludes the shuffled queue or possibly its the queue of the playlist playing. my counter shows 4 hours even though i dont have nearly enough songs that i manually added to me queue. a bit confusing as i thought it counts the time left in the queue i manually made

**Comment by Theblockbuster1** [comment]
Yeah, the timer basically shows how long until the music will stop playing (ignoring if you're on repeat mode). In #5 I've considered making an option to customise the timer to optionally show only how much time is left in your manual queue, excluding songs from the album or playlist you're listening from. I haven't done it yet because I didn't manage to implement the feature in the way that I wanted lol.
